sample_factory/enjoy.py

```diff
@@ -183,11 +185,11 @@ def max_frames_reached(frames):
                         finished_episode[agent_i] = True
                         rew = episode_reward[agent_i].item()
                         episode_rewards[agent_i].append(rew)
-                        if cfg.use_record_episode_statistics:
-                            if "episode" in infos[agent_i].keys():
-                                reward_list.append(infos[agent_i]["episode"]["r"])
-                        else:
-                            reward_list.append(rew)
+                        # if cfg.use_record_episode_statistics:
```

Comment:
Why did you comment this out?
I think we need this for Atari (@wmFrank knows the details)